cmd/proxy/config/config.toml

```diff
@@ -42,16 +42,16 @@
 # shard id 4294967295
 [[Observers]]
    ShardId = 0
-   Address = "http://127.0.0.1:8080"
+   Address = "http://127.0.0.1:8081"
```

Comment:
rollback these changes

Reply:
if you start proxy as is with that configuration you will have a loop. proxy starts at `localhost:8080` and you have and observer with `localhost:8080` and you will have a infinite loop of requests.

Reply:
ok. good catch.